components/gitpod-protocol/src/protocol.ts

```diff
@@ -120,6 +120,9 @@ export namespace User {
     }
 
     export function isOnboardingUser(user: User) {
+        if (!!user.organizationId) {
```

Comment:
:bulb: Would be great to resort to a method `User`, e.g. `User.isOrganizationalUser` or so. Just to highlight the dependency.

But this works, to unblocking nonetheless!